stdpopsim/drosophila_melanogaster.py

```diff
@@ -0,0 +1,115 @@
+"""
+Genome, genetic map and demographic model definitions for humans.
+"""
+
+import msprime
+
+import stdpopsim.models as models
+import stdpopsim.genomes as genomes
+import stdpopsim.genetic_maps as genetic_maps
+
+
+###########################################################
+#
+# Genetic maps
+#
+###########################################################
+
+
+class Comeron2012_dm6(genetic_maps.GeneticMap):
+    """
+    Comeron et al. (2012) maps (lifted over to dm6) used in
+    Currently needs a readme as to the lift over, etc.
+    """
+    url = (
+        "http://sesame.uoregon.edu/~adkern/dmel_recombination_map/"
+        "comeron2012_maps.tar.gz")
+    file_pattern = "genetic_map_comeron2012_dm6_{name}.txt"
+
+
+genetic_maps.register_genetic_map(Comeron2012_dm6())
+
+###########################################################
+#
+# Genome definition
+#
+###########################################################
+
+# List of chromosomes. Data for length information based on DM6,
+# https://www.ncbi.nlm.nih.gov/genome/?term=drosophila+melanogaster.
+# FIXME: add mean mutation and recombination rate data to this table.
+_chromosome_data = """\
+chrX   23542271
+chr2L   23513712
+chr2R   25286936
+chr3L   28110227
+chr3R   32079331
+chr4   1348131
+chrY   3667352
+chrM   19524
+"""
+
+_chromosomes = []
+for line in _chromosome_data.splitlines():
+    name, length = line.split()[:2]
+    _chromosomes.append(genomes.Chromosome(
+        name=name, length=int(length),
+        mean_mutation_rate=8.4e-9,  # WRONG!, underestimate used in S&S
+        mean_recombination_rate=8.4e-9))  # WRONG, underestimate used in S&S!
+
+
+#: :class:`stdpopsim.Genome` definition for D. melanogaster. Chromosome length data is
+#: based on `dm6 <https://www.ncbi.nlm.nih.gov/assembly/GCF_000001215.4/>`_.
```

Comment:
I tweaked the documentation a bit here to have a link (this gets rendered on RTD). Is this link correct?

Reply:
yep this is the correct link